### ISSUE TITLE
 MeshCentral 0.8.63~0.8.65 . [Usage Problem] [Error Logs After  BackupConfig and reboot Service]

### ISSUE THREAD

**Issue by Jacky0400143** [Ylianst/MeshCentral#2801]
Hi, 
Separate issue From #2793
and manybe Somewhat related to another question #2800  

Using Server Console Command "backupconfig" After Reboot Service, Error Logs Shows Below

```
-------- 2021/6/18 上午10:27:41 ---- 0.8.65 --------

/home/XX/node_modules/meshcentral/db.js:1566
else if (parent.args.mariadb.database) { dbname = parent.args.mariadb.database; }
^

TypeError: Cannot read property 'database' of undefined
at Object.obj.getBackupConfig (/home/XX/node_modules/meshcentral/db.js:1566:38)
at WebSocket.processWebSocketData (/home/XX/node_modules/meshcentral/meshuser.js:1441:43)
at WebSocket.emit (events.js:198:13)
at Receiver.receiverOnMessage (/home/XX/node_modules/ws/lib/websocket.js:718:20)
at Receiver.emit (events.js:198:13)
at Receiver.dataMessage (/home/XX/node_modules/ws/lib/receiver.js:414:14)
at Receiver.getData (/home/XX/node_modules/ws/lib/receiver.js:346:17)
at Receiver.startLoop (/home/XX/node_modules/ws/lib/receiver.js:133:22)
at Receiver._write (/home/XX/node_modules/ws/lib/receiver.js:69:10)
at doWrite (_stream_writable.js:415:12)

"TypeError" is new issue in error logs today.
"ERROR: MeshCentral Intel(R) AMT server port 4433 is not available. " ,only show up when service after restart.
"Watchdog timer timeout " ,It often appears from time to time recently
```

Sorry for asking so many questions
Many Thanks.

**Comment by VINISHVV** [comment]
@Jacky0400143 - which database you are using?

**Comment by Jacky0400143** [comment]
I using System default NeDB，Because I see it seems that less than one thousand machines can still handle it, although the installation guide describes one hundred machines.

But my raspeberry pi OS has mariadb installed but it is not used

**Comment by VINISHVV** [comment]
Can you share your configuration ?

**Comment by Ylianst** [comment]
Working on a fix for this.

**Comment by Ylianst** [comment]
Submitted the fix for this. When using NeDB, the server console command "backupconfig" would crash the server. Will be fixed in MeshCentral v0.8.66.

![image](https://user-images.githubusercontent.com/1319013/122956200-a337c280-d335-11eb-8e74-c294104d113e.png)
